packages/react-tooltip/src/stories/TooltipStyled.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import { Tooltip, TooltipProps } from '../Tooltip';
+import { Button } from '@fluentui/react-button';
+import { SlideTextRegular } from '@fluentui/react-icons';
+import { makeStyles } from '@griffel/react';
+
+const getStyles = makeStyles({
+  toolTip: {
```

Comment:
```suggestion
  tooltip: {
```

nit: ["tooltip"](https://en.wiktionary.org/wiki/tooltip) is a word AFAIR

Reply:
hah...yeah, not sure why i wrote it that way